contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -527,7 +527,7 @@ impl InterchainTokenService {
             source_address: source_address.clone(),
```

Comment:
make PR title scope more accurate and readable, e.g. `fix(interchain-token-service): emit data_hash instead of data in InterchainTransferReceivedEvent`